src/pages/categorias/problemas.tsx

```diff
@@ -82,6 +83,21 @@ export function ListaProblemas() {
         }
       >
         <HStack spacing={2}>
+          <Tooltip
+            label="Voltar para Tutoriais"
```

Comment:
```suggestion
            label="Voltar para Categorias de Problema"
```
Quando coloco o mouse no botão aparece a mensagem errada
![image](https://github.com/fga-eps-mds/2023-1-schedula-front/assets/46847658/27d08543-bc4a-4f10-95c3-18f468835767)